### DISCUSSION TITLE
zero frequency asymptotes for hydrodynamic properties

### DISCUSSION THREAD

**Discussion by rgcoe** [sandialabs/WecOptTool#107] [Theory]
Related to #86, we are populating the zero frequency hydrodynamic components (e.g., `A(w=0) = ...`). We do not plan to use these components, as there is no wave excitation at `w=0`, but we would like to put reasonable values if possible for consistency.

Per [Newman](https://mitpress.mit.edu/books/marine-hydrodynamics-40th-anniversary-edition) (sec 6.17):

- `A(w=infty) -> 1/2 m`, where `m` is the double-body added mass

**Comment by cmichelenstrofer** [comment]
The above equation applies to motions on the horizontal plane. From looking at that section, it seems like there is no general way to obtain the zero-frequency added mass without solving the radiation problem numerically. For the radiation damping, Newman states that it vanishes for both limits for all cases. 

When the BEM data is given without the zero-frequency we should set those values to:
- added mass: NaN
- radiation damping: 0.0
- Froude-Krylov: hydrostatic stiffness
- diffraction: 0.0?

and add a message (`log.info`) indicating this was done. We could also add a warning *__if__* trying to solve with a wave that has a zero-frequency component.
